app/views/pages/api_release_notes.html.haml

```diff
@@ -99,7 +99,9 @@
               Decimal
               %code
                 quantity
-              of over 3 (3.01+) only (not required for LGFS)
+              of over 0 (not required for LGFS)
+              %sup{ style: "font-size: 10px"}
```

Comment:
minor: instead of using inline css, you could use one of the font override classes. see: https://design-system.service.gov.uk/styles/typography/#font-override-classes